securedrop_client/conversation/transcript/templates/transcript.txt.jinja

```diff
@@ -0,0 +1,16 @@
+{% if items|length <= 0 %}{% trans %}No messages.{% endtrans %}{% else %}
```

Comment:
Mostly a drive-by comment since I was curious how jinja2 was being used when reviewing the builder PR; ignore if this was already discussed :)

Personally this is pretty close to the boundary of not using jinja2 and doing plain string formatting, but my understanding is that this will grow more complex over time?

I would move the "No messages." block into Python though, it adds an extra level of nesting that I don't think is necessary here.